practice-one/src/assets/fonts/font.css

```diff
@@ -0,0 +1,10 @@
+@font-face {
+  font-family: 'Roboto';
+  src: url('Roboto-ThinItalic-webfont.eot');
```

Comment:
Does this fonts work? I assumed you have to download these font files and then import them by URLs? I don't see any font file here?

Reply:
font.css should be in styles/base/